server/src/helpers/passwordCompare.ts

```diff
@@ -0,0 +1,15 @@
+import bcrypt from 'bcryptjs';
+
+const passwordCompare = (password:string, hash:string) => {
+  const isTrue = new Promise((resolve, reject) => {
+    bcrypt.compare(password, hash, (err, res) => {
```

Comment:
```suggestion
    bcrypt.compare(password, hash, (err, res) => {
```
// no need to convert to promise as bcrypt supports promise by nature